packages/graphql-cli-graphback-plugin/README.md

```diff
@@ -0,0 +1,36 @@
+## GraphBack
+
+<p align="center">
+  <img width="400" src="https://github.com/aerogear/graphback/raw/master/website/static/img/graphback.png">
+  <br/>
+  Auto generate database structure, <br/>
+  GraphQL Resolvers and Queries from GraphQL types 🚀
+</p>
+
+## Motivation 
+
+GraphQL can be hard and intimidating initially.
+Graphback helps you to kickstart your experience with any existing GraphQL implementation
+by generating backend layer using your data model.
+
+## Commands
+
+To generate backend using Graphback:
+
+```sh
+graphql backend init <name>
```

Comment:
should this be `graphql backend-init <name>` ?

Reply:
following Graphql-Cli format init is without `-` and other commands are with '-' but I can change it if you think it's unnecessary.

Reply:
I think we need to have subcomand as `backend` adding many commands for single extension is antipattern and it will not be merged to cli.